fe/fe-core/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java

```diff
@@ -537,6 +538,13 @@ private RollupJobV2 createMaterializedViewJob(String mvName, String baseIndexNam
                     }
                 }
             }
+            if (KeysType.UNIQUE_KEYS == keysType && olapTable.hasSequence()) {
+                if (meetValue) {
+                    // add the sequence column
+                    rollupSchema.add(keysNumOfRollup, new Column(olapTable.sequenceColName, olapTable.getSequenceType(),
+                            false, AggregateType.MAX, null, ""));
```

Comment:
why do not check if sequenceColName ia alread exist in rollupSchema

Reply:
add check Done